ruby-kafka.gemspec

```diff
@@ -25,6 +25,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency 'gssapi', '>=1.2.0'
```

Comment:
Yeah – but won't they be required by all `ruby-kafka` installations, since you're unconditionally requiring the library?